cli/command_repository_status.go

```diff
@@ -99,6 +110,16 @@ func (c *commandRepositoryStatus) run(ctx context.Context, rep repo.Repository)
 	ci := dr.BlobReader().ConnectionInfo()
 	c.out.printStdout("Storage type:        %v\n", ci.Type)
 
+	switch cp, err := dr.BlobVolume().GetCapacity(ctx); {
+	case err == nil:
+		c.out.printStdout("Storage capacity:    %v\n", units.BytesStringBase2(int64(cp.SizeB)))
```

Comment:
use base10 (GiB) here. Base2 (GB) is only historically used for RAM sizes.

Reply:
Done